Template10 (Validation)/Validation/ValidatableViewModelBase.cs

```diff
@@ -9,16 +10,17 @@
 
 namespace Template10.Validation
 {
-    public abstract class ModelBase : BindableBase, IModel
+    public abstract class ValidatableViewModelBase : ViewModelBase, IModel
     {
-        public ModelBase()
+        public ValidatableViewModelBase()
```

Comment:
He said this in the previous comment on the file User.cs. If it's that I don't care.